docs/starting_server.md

```diff
@@ -1,97 +1,144 @@
+Note that changing configuration in runtime while serving a single model is not possible. Serving multiple models requires a configuration file that stores settings for all served models. 
+When deploying model(s) with a configuration file, you can add or delete models, as well as update their configurations in runtime, without needing to restart the server.
 
-[Learn more](docker_container.md)
-
-## Running on Bare Metal and Virtual Machine (VM) Hosts
-
-OpenVINO Model Server is an open-source project written in C++. Therefore you can download and compile the code to obtain the binary and run it on bare metal.
-The `make` targets are provided to simplify the process.
-
-[Learn more](host.md)
-
-## Configuring Deployments
-
-Depending on performance requirements, traffic expectations, and what models OVMS serves, you may want to make certain adjustments to:  
-configuration of server options, like:
-- ports used
-- enable/disable REST API
-- set configuration monitoring 
-
-configuration for each of the served models, like:  
-- the device to load the model onto
-- the model version policy
-- inference related options
-
-[Learn more](parameters.md)
-
-## Keeping Models in a Remote Storage
+## Serving a Single Model
 
-Leverage remote storages, compatible with Google Cloud Storage (GCS), Amazon S3, or Azure Blob Storage, to create more flexible model repositories 
-that are easy to use and manage, for example, in Kubernetes deployments.
+Before starting the container, make sure you have [prepared the model for serving](models_repository.md).
 
-[Learn more](using_cloud_storage.md)
+Start the model server by running the following command with your parameters: 
 
-## Setting Model Versioning Policies for served models
+```
+docker run -d --rm -v <models_repository>:/models -p 9000:9000 -p 9001:9001 openvino/model_server:latest \
+--model_path <path_to_model> --model_name <model_name> --port 9000 --rest_port 9001 --log_level DEBUG
+```
 
-Take advantage of the model repository structure. Add or delete version directories and Model Server will automatically adjust. 
-Take full control over the served model versions by setting a model version policy and serving all, the chosen, or just the latest version of the model.
+Example using a ResNet model:
 
-[Learn more](model_version_policy.md)
+```bash
+mkdir -p models/resnet/1
+wget -P models/resnet/1 https://storage.openvinotoolkit.org/repositories/open_model_zoo/2022.1/models_bin/2/resnet50-binary-0001/FP32-INT1/resnet50-binary-0001.bin
+wget -P models/resnet/1 https://storage.openvinotoolkit.org/repositories/open_model_zoo/2022.1/models_bin/2/resnet50-binary-0001/FP32-INT1/resnet50-binary-0001.xml
 
-## Modifying Model Configuration in Runtime
+docker run -d --rm -v ${PWD}/models:/models -p 9000:9000 -p 9001:9001 openvino/model_server:latest \
+--model_path /models/resnet/ --model_name resnet --port 9000 --rest_port 9001 --log_level DEBUG
+```
```

Comment:
Can you change rest port number to 8000? This way it will we consistent with the rest of documentation files